squirrel_attention/src/fixate_on_3D_point.cpp

```diff
@@ -0,0 +1,77 @@
+#include <math.h>
+
+#include <ros/ros.h>
+#include <tf/transform_listener.h>
+
+#include "std_msgs/Float64.h"
+#include <dynamixel_controllers/SetRelativePosition.h>
```

Comment:
dynamixel_controllers? do we still need this?

Reply:
No, I will remove it in the next update.